src/Sources/EquationTypes.jl

```diff
@@ -0,0 +1,17 @@
+"""
+Types for dispatching on different equation sets for sources
+"""
+
+module EquationTypes
+
+export AbstractStyle
+export CoalescenceStyle
+export OneModeCoalStyle
+export TwoModesCoalStyle
+
+abstract type AbstractStyle end
+abstract type CoalescenceStyle <: AbstractStyle end
+struct OneModeCoalStyle <: CoalescenceStyle end
+struct TwoModesCoalStyle <: CoalescenceStyle end
+
```

Comment:
Can you explain the purpose of specifying OneMode and TwoModes? Are these implicitly analytical solutions, or would "analytical/numerical" be an additional subtype of each of these?

Reply:
I defined these to use multiple dispatch in Julia. In `Sources.jl` there are two coalescence functions. First one is for computing coalescence when you have defined a single size distribution. This one exist on the main branch. The second one is what I added for when we have to separate size distributions (like liquid and rain). This is also analytical (except one simple 1D integral that has to be computed numerically, for auto-conversion). In the future we might be able to unify these functions. But for now I decided to not touch the existing function and add mine as a separate function that can be called via Julia's multiple dispatch. In the future we should add the subtypes analytical/numerical and if possible unify the current functions under analytical and get rid of these "OneMode/TwoModes" subtypes.